lib/game.rb

```diff
@@ -0,0 +1,61 @@
+# Understands how to move through the big steps of tic-tac-toe.
```

Comment:
Domain design: Would it be possible for this description to be clearer on a specific domain concept rather than "big steps"?